applications/datalinker/values.yaml

```diff
@@ -10,16 +10,28 @@ image:
   repository: "ghcr.io/lsst-sqre/datalinker"
 
   # -- Pull policy for the datalinker image
-  pullPolicy: "IfNotPresent"
+  pullPolicy: "Always"
 
   # -- Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: "tickets-DM-51412"
```

Comment:
I usually only do this in the `values-idfdev.yaml` file to make sure I don't ever accidentally queue the development version up for prod.

Reply:
Yea good point